src/energyplus/ForwardTranslator.cpp

```diff
@@ -678,7 +678,7 @@ namespace energyplus {
       this->createStandardOutputRequests();
     }
 
-    Workspace workspace(StrictnessLevel::None, IddFileType::EnergyPlus);
+    Workspace workspace(StrictnessLevel::Minimal, IddFileType::EnergyPlus);
```

Comment:
Do you need a way to go to `None` here? either via checking if model's strictness is None => None, or via a ft.setStrictnesslevel method or something

Maybe you don't, I'm not 100% sure.